recursive-list-main/src/components/file/file.component.tsx

```diff
@@ -0,0 +1,24 @@
+import { LazyLog, ScrollFollow } from "react-lazylog";
+import { useFileStyles } from "./file.styles";
+
+export interface FileProps {
```

Comment:
you can create new file called file.interfaces.ts and move this interface there so that it can be reused.

Reply:
Created separate file.interfaces.ts for every component